src/Microsoft.DotNet.Darc/DarcLib/LocalLibGit2Client.cs

```diff
@@ -35,13 +38,17 @@ public LocalLibGit2Client(
     {
         _remoteTokenProvider = remoteTokenProvider;
         _processManager = processManager;
+        _exponentialRetry = new ExponentialRetry(Options.Create(new ExponentialRetryOptions
```

Comment:
I cheated a bit here, since this class is initialized with a constructor in a number of places. We could also properly register `ExponentialRetry` in the DI container, and create it like this in the other places using the constructor, but I don' think it matters too much..